### PR TITLE
New updates released.

### PR DESCRIPTION
**Pull request by erennuygun**
Here are the improvements that have been made.

- The web interface has been improved.

- A new index page has been created to add a response code and upload a static file.

- A referral page has been created using Ajax.

### CODE REVIEW

File: kodama.py [hunk 1]
```diff
@@ -47,38 +74,33 @@ def echo():
...
     source = FakeSocket(rawResponseEnc)
     parsedResponse = HTTPResponse(source)
     parsedResponse.begin()
     parsedResponse_body = parsedResponse.read()
     parsedResponse_headers = parsedResponse.getheaders()
     parsedResponse_status = parsedResponse.status
     parsedResponse.close()
-    
     response = make_response(parsedResponse_body)
+    response.set_cookie("mycookie", value="htegergergegf",secure=True,httponly=True)
```
Comment (ghurkan) [inline kodama.py]: This should be removed

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Kodama - Response it !</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/jpg" href="https://api.media.atlassian.com/file/7195568a-64ce-49f1-9279-8b5ee0ba7d98/binary?token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI5MjkwZTQzOS1hNDZkLTRmMjctODVjNC0yMWYyZGNmOTcxMjYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOjcxOTU1NjhhLTY0Y2UtNDlmMS05Mjc5LThiNWVlMGJhN2Q5OCI6WyJyZWFkIl19LCJleHAiOjE2MzY4MjY0MDYsIm5iZiI6MTYzNjc0MzQyNn0.z6mP04oBGA73F6cyBKOF6eaaEs317sqhHnjIhz_3hRI&client=9290e439-a46d-4f27-85c4-21f2dcf97126&name=ES&max-age=2940"/>
```
Comment (ghurkan) [inline templates/base.html]: This returns 403.

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            $('#uploadSuccess').css('visibility', 'visible');
+        });
+    });
+}); 
+</script>
+<h1>{{ the_title }}</h1>
+
+<div class="container">
+    <form  method="POST">
+        <img style="display: block;" class="topright" src="https://dpsvdv74uwwos.cloudfront.net/statics/img/blogposts/invicti-security-family-of-products.png" height="200" width="365" id="img">
```
Comment (ghurkan) [inline templates/index.html]: Please remove company mentions

File: templates/index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html' %}
+
+{% block body %}
+
+<script src="/static/jquery.min.js"></script>
```
Comment (ghurkan) [inline templates/index.html]: This is returning 404

File: kodama.py (hunk 1 above)
Comment (erennuygun) [reply kodama.py]: Removed.

File: templates/base.html (hunk 2 above)
Comment (erennuygun) [reply templates/base.html]: Deleted because it mentions company.

File: templates/index.html (hunk 3 above)
Comment (erennuygun) [reply templates/index.html]: Deleted because it mentions company.

File: templates/index.html (hunk 4 above)
Comment (erennuygun) [reply templates/index.html]: Deleted because it's not necessary.

**Pull request merged**